src/main/java/org/openjdk/jextract/clang/CursorKind.java

```diff
@@ -100,8 +100,8 @@ public enum CursorKind {
      * Per libclang API docs, clang returns this CursorKind
      * for both C11 _Static_assert and C++11 static_assert
      */
-    StaticAssert(CXCursor_StaticAssert());
-
+    StaticAssert(CXCursor_StaticAssert()),
+    Unsupported(-1);
```

Comment:
I've added this, otherwise jextract will crash every time it sees a cursor it doesn't understand, as `Parser` calls `getKind` on the cursors it sees. Alternatives would be to (a) add an enum constant here for all the cases libclang support, or (b) skip this enum entirely, and just use jextract constants. I've held off a bigger refactoring for now, but this feels an area where we seem to be creating more work for ourselves than needed.